README.md

```diff
@@ -141,10 +141,18 @@ docker run --rm -e REACT_APP_HOME_SERVER_URL=https://matrix-client.matrix.org -p
 
 We also provide a [HELM chart](./charts/).
 
-> Important:
-> The collaboration features require you to configure TURN servers in your Homeserver.
-> It can work without them, but there is a high chance that it won't!
-> The official matrix.org homeserver will work; see for example how to configure them for [Synapse](https://matrix-org.github.io/synapse/latest/turn-howto.html).
+### Important notes
```

Comment:
I feel like this info should not only be under deployment, because it is possible to just use the widget (e.g. the demo instance further up) and therefore never read this. Users should still be aware of these issues in these cases, don't you agree? 🤔

Reply:
Same applies to turn servers then. Should we just make the import notes a top level headline?